edgelet/aziot-edged/Cargo.toml

```diff
@@ -4,6 +4,9 @@ version = "0.1.0"
 authors = ["Azure IoT Edge Devs"]
 publish = false
 edition = "2021"
+description = "The aziot-edged is the IoT edge daemon."
+homepage = "https://github.com/azure/iotedge"
```

Comment:
```suggestion
homepage = "https://aka.ms/iotedge"
```

Reply:
_HOMEPAGE
Website where more information about the software the recipe is building can be found._
https://docs.yoctoproject.org/1.6/bitbake-user-manual/bitbake-user-manual.html

Reply:
The aziotd and aziotctl recipes are pointing to a _documentation_ page rather than directly to the github repo.  Seems we should do the same for the IoT Edge recipes.  And in that case the homepage that I think would serve well as a starting point to learn about the software is the one on the MS Learn site.